submissions/Barasii/HTML-CSS-Popup/css/style.css

```diff
@@ -0,0 +1,313 @@
+* {
+    margin: 0;
+    padding: 0;  
+}
+
+header,main,nav,footer, {
+    display: block;
+}
+
+body {
+    font-family: arial,sans-serif;
+    font-size: 1rem;
+    font-weight: 500;
+    line-height: 1.5;
+    color: #222;
+    background-color: #fff;
+}
+
+ul {
+    list-style: none;
+    list-style-type: none;
```

Comment:
`list-style` is a [shorthand property](https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties). Don't need `list-style-type` + `list-style` here. Usually `list-style-type` will be enough.